### PR TITLE
Updating Sailfish OS

### PR DESCRIPTION
**Pull request by jovirkku**
Signed-off-by: Jorma Virkkunen <jorma.virkkunen@jolla.com>

### CODE REVIEW

**Comment by jovirkku** [comment]
Please review now.

File: Support/Help_Articles/Updating_OS/README.md [hunk 1]
```diff
@@ -0,0 +1,404 @@
+---
+title: How to update Sailfish OS
+permalink: Support/Help_Articles/Updating_OS/
+parent: Help Articles
+layout: default
+nav_order: 90
```
Comment (vigejolla) [inline Support/Help_Articles/Updating_OS/README.md:6]: Why 90? I would have picked 50, but perhaps you have a plan about other pages to be added here.

File: Support/Help_Articles/Updating_OS/README.md [hunk 2]
```diff
@@ -0,0 +1,404 @@
+---
+title: How to update Sailfish OS
+permalink: Support/Help_Articles/Updating_OS/
```
Comment (vigejolla) [inline Support/Help_Articles/Updating_OS/README.md:3]: Let's try to keep the titles and directory names identical, it makes it easier to find the pages in the tree

Comment (jovirkku) [reply Support/Help_Articles/Updating_OS/README.md:3]: My idea was that while the permalink should be fairly compact, the title could be more verbose.

File: Support/Help_Articles/Updating_OS/README.md (hunk 1 above)
Comment (jovirkku) [reply Support/Help_Articles/Updating_OS/README.md:6]: 50 would be better. Leave room for more documents (in alphabetical order if we want that)

File: Support/Help_Articles/Updating_OS/README.md (hunk 2 above)
Comment (vigejolla) [reply Support/Help_Articles/Updating_OS/README.md:3]: There's no reason for keeping the permalinks a.k.a. directory names compact. See e.g. [Upstream_Git_with_Long-Lived_Topic_Branch_Approach](https://github.com/sailfishos/docs.sailfishos.org/blob/master/Develop/Platform/Usage_of_packaging_formats/Upstream_Git_with_Long-Lived_Topic_Branch_Approach/README.md?plain=1#L3) for an example of a fairly long permalink. That being said, I think "Updating Sailfish OS" would be a good middleground.

File: Support/Help_Articles/Updating_OS/README.md (hunk 1 above)
Comment (jovirkku) [reply Support/Help_Articles/Updating_OS/README.md]: permalink: Support/Help_Articles/Updating_Sailfish_OS/
nav_order: 150   (alphabetical order)

Done.

File: Support/Help_Articles/Updating_OS/README.md (hunk 2 above)
Comment (jovirkku) [reply Support/Help_Articles/Updating_OS/README.md]: New title: Updating Sailfish OS

**Comment by jovirkku** [comment]
The new framework for the pictures is waiting for the instructions.

File: Support/Help_Articles/Updating_Sailfish_OS/README.md [hunk 3]
```diff
@@ -0,0 +1,405 @@
...
+
+
+
+# What if installing an OS update fails (but download worked)
+
+_Sometimes installing an OS update may fail even though the download seemingly worked okay. The problem is usually caused by the device and Jolla Store being out of sync somehow._
+
+The picture below indicates a failure with the OS installation. Restart the phone and try again.
+
+|<a href="OS-could-not-be-updated-reboot.jpg" style="width:25em;display:block"><img src="OS-could-not-be-updated-reboot.jpg" alt="Adding an account" class="md_thumbnail" style="max-width:100%"/></a>|
```
Comment (sledges) [inline Support/Help_Articles/Updating_Sailfish_OS/README.md:289]:
```suggestion
|<a href="OS-could-not-be-updated-reboot.jpg" style="width:25em;display:block"><img src="OS-could-not-be-updated-reboot.jpg" alt="OS could not be updated - reboot" class="md_thumbnail" style="max-width:100%"/></a>|
```

File: Support/Help_Articles/Updating_Sailfish_OS/README.md [hunk 4]
```diff
@@ -0,0 +1,405 @@
...
+
+In case of trouble, we would ask you to run the command below and send a picture (pictures) showing the complete printout. Then file a **[service request](https://jolla.zendesk.com/hc/en-us/requests/new)** and attach the pictures to it.
+
+```
+ssu lr
+```
+
+The correct output should be something like in the picture below (an example from Xperia XA2):
+
+|<a href="Example-ssu-lr-xa2.png" style="width:50em;display:block"><img src="Example-ssu-lr-xa2.png" alt="Adding an account" class="md_thumbnail" style="max-width:100%"/></a>|
```
Comment (sledges) [inline Support/Help_Articles/Updating_Sailfish_OS/README.md:375]:
```suggestion
|<a href="Example-ssu-lr-xa2.png" style="width:50em;display:block"><img src="Example-ssu-lr-xa2.png" alt="Example ssu lr" class="md_thumbnail" style="max-width:100%"/></a>|
```

File: Support/Help_Articles/Updating_Sailfish_OS/README.md [hunk 5]
```diff
@@ -0,0 +1,405 @@
...
+|-|
+|<span class="md_figcaption">OS installation failed</span>|
```
Comment (sledges) [inline Support/Help_Articles/Updating_Sailfish_OS/README.md:377]:
```suggestion
|<span class="md_figcaption">Output of `ssu lr`</span>|
```

**Review by sledges — CHANGES_REQUESTED** [review]
Small bits, most can be corrected when migrating to #138.

**Comment by vigejolla** [comment]
Looks good, but please rebase on top of master

File: Support/Help_Articles/Updating_OS/README.md (hunk 1 above)
Comment (vigejolla) [reply Support/Help_Articles/Updating_OS/README.md]: I don't really care what the order is, but... In my alphabet U(pdating Sailfish OS) is after T(ips and Tricks). But the nav_order 1000 in "Tips and Tricks" hints that perhaps it is supposed to be after all the other pages, which should be in alphabetical order? Like I said, I don't care about the order, I just want to be sure the end result is what you want.

**Review by vigejolla — APPROVED** [review]
LGTM. Just one question about the order of pages (to make sure). After that I can merge this.

**Comment by jovirkku** [comment]
> LGTM. Just one question about the order of pages (to make sure). After that I can merge this.

I am not sure if there is a question for me. I have tried to get the docs under Help Articles in alphabetical order (in the lack of any guidance).

**Comment by vigejolla** [comment]
> I am not sure if there is a question for me. I have tried to get the docs under Help Articles in alphabetical order (in the lack of any guidance).

Yes, there was a question, as I wasn't sure if that was indeed what you wanted. And since the pages are *not* in alphabetical order, I suppose there is something left to fix. e.g. increase nav_order to something above 1000.

**Pull request merged**